### PR TITLE
11 dev add ctdirectjl test problems

### PR DESCRIPTION
**Pull request by 0Yassine0**

### CODE REVIEW

**Comment by ocots** [comment]
@PierreMartinon est-ce qu'on ajoute d'autres problèmes cf l'issue liée ?

File: Project.toml [hunk 1]
```diff
@@ -5,6 +5,7 @@ version = "0.1.0"
 
 [deps]
 CTBase = "54762871-cc72-4466-b8e8-f6c8b58076cd"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
```
Comment (frapac) [inline Project.toml:8]: I would prefer we remove DataFrames.jl from the deps

File: ext/JuMPModels/bioreactor.jl [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    beta = 1
+    c = 2
+    gamma = 1
+    halfperiod = 5
+    Ks = 0.05
+    mu2m = 0.1
+    mubar = 1
+    r = 0.005
+    T = 10 * N
+    pi = 3.141592653589793
```
Comment (frapac) [inline ext/JuMPModels/bioreactor.jl:16]: You can use `pi` without defining it in Julia.

File: ext/JuMPModels/insurance.jl [hunk 3]
```diff
@@ -0,0 +1,70 @@
+"""
+The Insurance Problem:
+    The problem is formulated as a JuMP modeln and can be found [here](https://github.com/control-toolbox/bocop/tree/main/bocop)
```
Comment (frapac) [inline ext/JuMPModels/insurance.jl:3]: typo: `modeln`

File: ext/JuMPModels/bioreactor.jl [hunk 4]
```diff
@@ -0,0 +1,58 @@
+"""
+The Bioreactor Problem:
+    The problem is formulated as a JuMP modeln and can be found [here](https://github.com/control-toolbox/bocop/tree/main/bocop)
```
Comment (frapac) [inline ext/JuMPModels/bioreactor.jl:3]: typo: `modeln`

File: ext/JuMPModels/beam.jl [hunk 5]
```diff
@@ -0,0 +1,39 @@
+"""
+The Beam Problem:
+    The problem is formulated as a JuMP modeln and can be found [here](https://github.com/control-toolbox/bocop/tree/main/bocop)
```
Comment (frapac) [inline ext/JuMPModels/beam.jl:3]: typo: `modeln`

File: ext/JuMPModels/jackson.jl [hunk 6]
```diff
@@ -0,0 +1,47 @@
+"""
+The Jackson Problem:
+    The problem is formulated as a JuMP modeln and can be found [here](https://github.com/control-toolbox/bocop/tree/main/bocop)
```
Comment (frapac) [inline ext/JuMPModels/jackson.jl:3]: typo: `modeln`

File: ext/JuMPModels/robbins.jl [hunk 7]
```diff
@@ -0,0 +1,45 @@
+"""
+The Robbins Problem:
+    The problem is formulated as a JuMP modeln and can be found [here](https://github.com/control-toolbox/bocop/tree/main/bocop)
```
Comment (frapac) [inline ext/JuMPModels/robbins.jl:3]: typo: `modeln`

File: ext/OptimalControlModels/beam.jl [hunk 8]
```diff
@@ -0,0 +1,27 @@
+"""
+The Beam Problem:
+    The problem is formulated as an OptimalControl modeln and can be found [here](https://github.com/control-toolbox/bocop/tree/main/bocop)
```
Comment (frapac) [inline ext/OptimalControlModels/beam.jl:3]: typo: `modeln`

File: src/OptimalControlProblems.jl [hunk 9]
```diff
@@ -1,6 +1,7 @@
 module OptimalControlProblems
 
 using CTBase
+using DataFrames
```
Comment (frapac) [inline src/OptimalControlProblems.jl:4]: I would remove DataFrames here

**Review by frapac — CHANGES_REQUESTED** [review]

File: Project.toml (hunk 1 above)
Comment (ocots) [reply Project.toml:8]: But he needs it for the metadata. What do you suggest?

Comment (ocots) [reply Project.toml:8]: Actually, he did the same as here:

https://github.com/JuliaSmoothOptimizers/OptimizationProblems.jl/blob/2c14b0296eebae606a86edcbcfebb2b7faa0bb21/src/OptimizationProblems.jl#L84

File: ext/JuMPModels/beam.jl (hunk 5 above)
Comment (ocots) [reply ext/JuMPModels/beam.jl:3]: Done!

File: Project.toml (hunk 1 above)
Comment (frapac) [reply Project.toml]: I would just use dictionary to store the metadata, I don't think we need to use DataFrames for that

**Review by frapac — APPROVED** [review]

**Pull request merged**